### DISCUSSION TITLE
LGT8F328P available in QFN32 package

### DISCUSSION THREAD

**Discussion by LaZsolt** [dbuezas/lgt8fx#294] [Show and tell]
I found it on Aliexpress, LGT8F328P in QFN32 package:

| BTE23-13 | BTE23-08 |
| - | - |
| ![LGT8F328P-QFN32-MiniEVB-800x500](https://github.com/dbuezas/lgt8fx/assets/60486174/6a7bb068-e86e-4ec3-97e9-7645bcc482f0) | ![LGT8F328P-QFN32-MiniEVB2-850x500](https://github.com/dbuezas/lgt8fx/assets/60486174/0b0b5429-72a4-4de6-b310-34e707ada6ab) |

![LGT8F328P-QFP32-QFN32-QFP48](https://github.com/dbuezas/lgt8fx/assets/60486174/d3a18ffd-6167-4356-b79a-ac831a8c7674)

**Comment by dbuezas** [comment]
Very neat! counting pin, it is the same board size as the SSOP20 but with an external clock and more exposed pins.
How long until @wollewald comes out with a pinout diagram? 😀

**Reply by wollewald** [reply]
I searched for these boards on AliExpress, found them after some time, but unfortunately none of them were available for shipping to Germany. 😞

**Reply by LaZsolt** [reply]
Only 3 left at this moment. Here is a link: https://www.aliexpress.com/i/1005004884270856.html
The other board: https://www.aliexpress.com/i/33003833124.html

**Reply by LaZsolt** [reply]
When I set the shipping address to Germany then the item became unavailable.
I can send it to you if you want after I receceive it.

**Reply by wollewald** [reply]
That would be very kind! Thank you so much for your offer.

**Comment by trimarco232** [comment]
for those boards, the only new things is the crystal , ie. pin3(PE4/0C0A* & PF4/OC1B*/ICP3) and pin6(PE5/AC1O/CLKO & PF5/OC1A*) are still lost ...

**Comment by trimarco232** [comment]
one have to take care that both boards are available (when they are) in both 5v or 3v3 versions

**Comment by wollewald** [comment]
Now I have a BT13 module (thanks @LaZsolt!) and I tried a bit. From what I have seen by now, it behaves like a QFP32 that is mounted on a SSOP20 MiniEVB board. E.g. the IC itself has separate PD3 and PD4 pins which are connected to the one 3/4 board pin. And in contrast to the SSOP20 MiniEVB PF4 ist not available. So, as @trimarco232 already stated the advantage of this board is the crystal. Not sure if it is worth to create another pinout scheme for this module.
